express-app/public/css/my.css

```diff
@@ -21,7 +18,19 @@ body {
 }
 
 #input-family {
-  margin-bottom: 10px;
+  margin-top: 30px;
+  /*margin-bottom: 10px;*/
+}
+
+#create-members,
+#submit-income,
+#income-button,
+#estudios-btn {
+  margin-top: 20px; 
+}
+
+#add-member-button {
+  margin-bottom: 20px;
```

Comment:
@erikiado I don't agree, if this button needs more style we can add all of them there, also, at the view, it looks cleaner with just one id and not 2.